packages/vite/src/node/server/middlewares/transform.ts

```diff
@@ -50,7 +51,11 @@ export function transformMiddleware(
   } = server
 
   // Keep the named function. The name is visible in debug logs via `DEBUG=connect:dispatcher ...`
-  return async function viteTransformMiddleware(req, res, next) {
+  const viteTransformMiddleware: Connect.NextHandleFunction = async (
+    req,
+    res,
+    next,
+  ) => {
```

Comment:
Could we keep this as `async function viteTransformMiddleware(req, res, next) {` to avoid the diff in the change set? Having the name is also part of the API, I think it still work with the variable but it is more clear with the function form. And the same in the other middlewares

Reply:
There are ts errors on the function form. We have to add the ts type for each function param. This will still cause the diff in the change set.

Reply:
Did you check that the functions are correctly named?

Reply:
Yeah. I checked this 

https://github.com/senchalabs/connect/blob/master/index.js#L106

<img width="512" alt="image" src="https://user-images.githubusercontent.com/102238922/231654212-94c5d0d1-701b-49a3-804f-19315946f9a6.png">